src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -38,10 +38,14 @@
 
     private final StudyService studyService;
     private final ReportService reportService;
+    private final NetworkStoreService networkStoreService;
+    private final NetworkModificationService networkModificationService;
 
-    public StudyController(StudyService studyService, ReportService reportService) {
+    public StudyController(StudyService studyService, NetworkStoreService networkStoreService, NetworkModificationService networkModificationService, ReportService reportService) {
```

Comment:
Do you forgot to add an equipment search method to the controller?

Reply:
yes

Reply:
Done